lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -226,7 +226,7 @@ def __init__(self, *args, distro: Type[OpenSearchDistribution] = None):
             self.on[PeerRelationName].relation_joined, self._on_peer_relation_joined
         )
         self.framework.observe(
-            self.on[PeerRelationName].relation_changed, self._on_peer_relation_changed
+            self.on[PeerRelationName].relation_changed, self.peer_relation_changed
```

Comment:
I understand the reason behind the renaming (to be invoked by non juju events) but in order to make it clear that this is primarily a hook, could we add back the prefix `_on`?

Reply:
Yes, I was exposing it on some earlier iterations @Mehdi-Bendriss